connector/src/main/java/org/geysermc/connector/network/translators/java/JavaJoinGameTranslator.java

```diff
@@ -110,6 +110,9 @@ public void translate(GeyserSession session, ServerJoinGamePacket packet) {
             session.sendDownstreamPacket(new ClientPluginMessagePacket("minecraft:register", PluginMessageUtils.getFloodgateRegisterData()));
         }
 
+        // we can always do this. vanilla will just ignore it.
+        session.sendDownstreamPacket(new ClientPluginMessagePacket("minecraft:register", PluginMessageUtils.getEmoteRegisterData()));
```

Comment:
Don't send two register packets; send one of the same. See https://github.com/GeyserMC/Geyser/blob/godbridgingblock/connector/src/main/java/org/geysermc/connector/utils/PluginMessageUtils.java as an example.

Reply:
https://github.com/KosmX/Geyser/commit/cf524cbfcbf5a1d138ee11bc3ef36be77ddec39d
Is this a good solution?